compiler/rustc_error_messages/locales/en-US/trait_selection.ftl

```diff
@@ -0,0 +1,17 @@
+dump_vtable_entries = vtable entries for `{$trait_ref}`: {$entries}
```

Comment:
We prefer that these start with the crate name to avoid conflicts - we now check this in our macros this would likely fail CI after rebasing :)